pkg/reaper/podreaper/podreaper.go

```diff
@@ -217,3 +225,81 @@ func (ctx *ReaperContext) validateArguments(args *Args) error {
 
 	return nil
 }
+
+func loadFakeAPI(ctx *ReaperContext) {
```

Comment:
Why do we need this moved to `podreaper.go` it's the same package, we can keep this code under `podreaper_test.go`